core/src/test/java/org/infinispan/globalstate/NoOpGlobalConfigurationManager.java

```diff
@@ -1,38 +1,41 @@
 package org.infinispan.globalstate;
 
 import java.util.EnumSet;
+import java.util.concurrent.CompletableFuture;
 
 import org.infinispan.commons.api.CacheContainerAdmin;
 import org.infinispan.configuration.cache.Configuration;
 import org.infinispan.manager.EmbeddedCacheManager;
 import org.infinispan.test.TestingUtil;
+import org.infinispan.util.concurrent.CompletableFutures;
 
 /*
  * A no-op implementation for tests which mess up with initial state transfer and RPCs
  */
 public class NoOpGlobalConfigurationManager implements GlobalConfigurationManager {
    @Override
-   public Configuration createCache(String cacheName, Configuration configuration, EnumSet<CacheContainerAdmin.AdminFlag> flags) {
+   public CompletableFuture<Configuration> createCache(String cacheName, Configuration configuration, EnumSet<CacheContainerAdmin.AdminFlag> flags) {
       return null;
```

Comment:
Should these return a completed state to prevent NPEs?

Reply:
Done